legend-shared-pac4j-gitlab/src/main/java/org/finos/legend/server/pac4j/gitlab/GitlabPersonalAccessTokenCredentials.java

```diff
@@ -0,0 +1,89 @@
+// Copyright 2021 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.server.pac4j.gitlab;
+
+import org.apache.commons.lang.builder.EqualsBuilder;
+import org.apache.commons.lang.builder.HashCodeBuilder;
+import org.pac4j.core.credentials.Credentials;
+
+import java.util.Map;
+
+public class GitlabPersonalAccessTokenCredentials extends Credentials
+{
+    private String personalAccessToken;
+    private String userId;
+    private String state;
+    private String userName;
```

Comment:
Should these really be part of the credentials? Shouldn't the credentials just be the token?